app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/config/featureswitch/FeatureSwitch.scala

```diff
@@ -29,12 +29,18 @@ object FeatureSwitch {
   val prefix = "feature-switch"
 
   val switches: Set[FeatureSwitch] = Set(
-    TestFeature
+    TestFeature,
+    MandatedIncomeFeature
   )
   def get(str: String): Option[FeatureSwitch] = switches find (_.key == str)
 
   case object TestFeature extends FeatureSwitch {
     override val key = "test"
     override val displayText = "Test Feature"
   }
+
+  case object MandatedIncomeFeature extends FeatureSwitch {
+    override val key = "mandatedIncome"
+    override val displayText = "4 New Mandated Income"
```

Comment:
Could you remove the 4 please as we might re-use this feature switch for the next batch of income types we get